Source/Flow/Public/FlowComponent.h

```diff
@@ -203,9 +203,23 @@ class FLOW_API UFlowComponent : public UActorComponent
 	UFUNCTION(BlueprintPure, Category = "RootFlow", meta = (DeprecatedFunction, DeprecationMessage="Use GetRootInstances() instead."))
 	UFlowAsset* GetRootFlowInstance() const;
 
+//////////////////////////////////////////////////////////////////////////
+// UFlowComponent overrideable events
+
+public:
+	// Called when a Root flow asset triggers a CustomOutput
+	UFUNCTION(BlueprintImplementableEvent, DisplayName = "OnTriggerRootFlowOutputEvent")
+	void BP_OnTriggerRootFlowOutputEvent(UFlowAsset* RootFlowInstance, const FName& EventName);
+	virtual void OnTriggerRootFlowOutputEvent(UFlowAsset& RootFlowInstance, const FName& EventName) { }
+
+	//~Begin UFlowAsset-only access
+	FORCEINLINE void OnTriggerRootFlowOutputEventDispatcher(UFlowAsset& RootFlowInstance, const FName& EventName);
```

Comment:
Dispatcher calls both the blueprint and code overrides for the output trigger event